src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/Serializer/config.yml

```diff
@@ -6,9 +6,16 @@ framework:
         enabled: true
         default_context:
             enable_max_depth: true
+            # The option below is used in \Symfony\Bundle\FrameworkBundle\Tests\Functional\SerializerTest::testObjectNormalizerUsesDefaultContextConfigOption
+            # to assert that the `default_context` is taken into account in serializers
```

Comment:
The test itself should be enough to spot any issue quickly so I'm not sure this is useful

Reply:
You're right.
Maybe do I need to change the test to take into account the other normalizers that have the same behavior with `$defaultContext` since @nicolas-grekas has fixed them too?

Reply:
that's be nice if you can yes !

Reply:
Done.
I changed the tests to target all normalizers and encoders that are tagged `serializer.normalizer` or `serializer.encoder` and that have the argument `$defautContext` in their constructor, the same as what `SerializerPass` does.